plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -47,7 +47,7 @@ public function base_render_before($sender) {
      * Whitelist of allowed field properties.
      */
     public $FieldProperties = ['Name', 'Label', 'FormType', 'Required', 'Locked',
-        'Options', 'Length', 'Sort', 'OnRegister', 'OnProfile', 'OnDiscussion'];
```

Comment:
This should've been done via the cloud Salesforce addon, not put into core. Hard-coding references to other addons generally means something went wrong in our architecture planning.